packages/core/modules-sdk/src/remote-query/query.ts

```diff
@@ -20,6 +20,26 @@ import {
 export class Query {
   #remoteQuery: RemoteQuery
 
+  /**
+   * Method to wrap execution of the graph query for instrumentation
+   */
+  static traceGraphQuery?: (
+    queryFn: () => Promise<any>,
+    queryOptions: RemoteQueryObjectConfig<any>
+  ) => Promise<any>
+
+  /**
+   * Method to wrap execution of the remoteQuery overload function
+   * for instrumentation
+   */
+  static traceRemoteQuery?: (
+    queryFn: () => Promise<any>,
+    queryOptions:
```

Comment:
suggestion: should we rename the options to config? The options are used for other things

Reply:
The config has flags like `throwIfUndefined`. I think, they are not useful for tracing or do you think otherwise?